bot/exts/evergreen/realpython.py

```diff
@@ -0,0 +1,80 @@
+# Credit to Vthechamp22, who heavily inspired this bit of code
+import logging
+from html import unescape
+from urllib.parse import quote_plus
+
+from discord import Embed, HTTPException
+from discord.ext import commands
+
+from bot import bot
+from bot.constants import Colours
+
+logger = logging.getLogger(__name__)
+
+API_ROOT = "https://realpython.com/search/api/v1/?"
+ARTICLE_URL = "https://realpython.com{article_url}"
+SEARCH_URL = "https://realpython.com/search?q={user_search}"
+
+ERROR_EMBED = Embed(
+    title="Error while searching Real Python",
```

Comment:
I don't see a case where this is used, maybe remove it. Sorry if this is wrong, I'm reviewing code from my phone

Reply:
I mean ERROR_EMBED

Reply:
Line 40 is where it's used if an error comes up during the API call. I could be more specific about what kind of error, maybe HTTP_ERROR or RESPONSE_ERROR ?

Reply:
apologies. The name is fine iirc, you can change it if you want but I noticed you forgot to set the color, so there is some inconsistency with the other embeds, maybe consider red

Reply:
Ahh I did forget the color, just added it!